mindjunkies/forums/views.py

```
@@ -7,9 +7,10 @@
 from mindjunkies.courses.models import Course, Module
 
 from .forms import ForumCommentForm, ForumReplyForm, ForumTopicForm
-from .models import ForumComment, ForumTopic, Reply
-
+from .models import Course, ForumComment, ForumTopic, Reply
```

Ruff (0.8.2)

Redefinition of unused `Course` from line 7

(F811)